raspberry/bzzz/sensors/position_tracker_camera.py

```diff
@@ -0,0 +1,193 @@
+import numpy as np
+from PIL import Image
+import cv2
+from matplotlib import pyplot as plt
+from time import time_ns
+import numpy as np
+import math
+
+
+class Camera:
```

Comment:
Since this is part of the code that runs on Raspberry Pi, it should not be outputting anything. It should only track the position of an object.

Reply:
We should leave the code there for testing purposes. It will be helpful while setting up the camera and tweaking the code. Also, this is written assuming the camera is mounted on the ground, so we might want to have a view to what the camera is sending while changing the logic. We can remove it later.